build/Publish.targets

```diff
@@ -256,12 +251,12 @@
     <ItemGroup>
       <PackageToPublishToTransport
         Include="@(PackageToPublish)"
-        Condition="'%(PackageToPublish.Category)' == 'ship'" />
+        Condition="'%(PackagesToPublish.Category)' == 'ship'" />
```

Comment:
Can you revert this and the next one down? This would break publishing. We renamed in dev.